ui/src/dashboards/components/OverlayControls.js

```diff
@@ -25,19 +31,24 @@ const OverlayControls = props => {
             </li>
           ))}
         </ul>
-        <ConfirmButtons onCancel={onCancel} onConfirm={onSave} />
+        <ConfirmButtons
+          onCancel={onCancel}
+          onConfirm={onSave}
+          isSavable={isSavable}
```

Comment:
`<ConfirmButtons>` is looking for an `isDisabled` prop, so this will not pass through successfully.